frontends/web/src/components/alert/Alert.tsx

```diff
@@ -0,0 +1,98 @@
+/**
+ * Copyright 2018 Shift Devices AG
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Component, h } from 'preact';
+import { translate } from 'react-i18next';
+import Dialog from '../dialog/dialog';
+import { Button } from '../forms';
+
+let alertUser: (message: string, callback?: () => void) => void;
+
+interface Props {
+    t: () => void;
+}
+
+interface State {
+    active: boolean;
+    message?: string;
+}
+
+@translate()
+class Alert extends Component<Props, State> {
+    private button!: Component;
+    private callback!: () => void;
+
+    constructor(props) {
+        super(props);
+        alertUser = this.alerted;
+        this.state = {
+            active: false,
+        };
+    }
+
+    private setButtonRef = element => {
+        this.button = element;
+    }
+
+    private handleClose = e => {
```

Comment:
Please type `e` as `Event`, no import necessary, see https://github.com/digitalbitbox/bitbox-wallet-app/blob/f4309109de5cf8392d2aba990e1798b6c041ff9c/frontends/web/src/components/fiat/fiat.tsx#L33-L37

Reply:
Or since you don't use `e` at all, maybe just write `() => {…}`? JS is definitely happy with this and so should TS if I didn't make any mistakes: https://www.typescriptlang.org/play/index.html#src=const%20f%20%3D%20()%20%3D%3E%20%7B%20%7D%3B%0D%0Aconst%20g%3A%20(e%3A%20Event)%20%3D%3E%20void%20%3D%20f%3B%0D%0A